LibraryManagementSystem/Forms/ViewTab.cs

```diff
@@ -35,7 +35,7 @@ private void ViewTab_Load(object sender, EventArgs e)
         {
             // Focus on datagrid because focusing on searchbox removes the appeal of placeholder
             this.ActiveControl = dgvBooks;
-            con = new SqlConnection("Data Source=MAJO-PC\\SQLEXPRESS;Initial Catalog=libraryData;Integrated Security=True");
+            con = new SqlConnection("Data Source=DESKTOP-9MBNT14\\SQLEXPRESS;Initial Catalog=libraryData;Integrated Security=True");
```

Comment:
Yes we should probably add a global variable to replace all these server names. I'll try it muna sa forked repo ko then I'll create a pull request if ever.